### PR TITLE
Create wmt-terminology.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
# Description

A new article that reflects the terminology used in the WMT conference results. 

## Type of PR

- Creates the article _[WMT Terminology]_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).
- [x] I have made sure that the URL is not already in use.
- [x] I have cross-linked relevant articles.
- [x] I have used the UK spelling.
- [x] I have kept consistency with the structure of sister articles in the same directory.

### CODE REVIEW

File: events/wmt-terminology.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+In this configuration, the human assessors rate the source input and system output without any reference translation.
+
+## Average score and z-score
+
+There are two main types of scores produced by WMT: average scores and z-scores.
+The average z-score is used as the official ranking of the conference results.
+The z-score shows how many standard deviations below or above the mean an average or raw score is.
+
+To compute the z-score,
+1. Human assessment scores for translations are standardized according to each human assessor’s overall mean and standard deviation score.
```
Comment (cefoo) [inline events/wmt-terminology.md:51]: Should we define `deviation score`?

File: events/wmt-terminology.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+parent: Events
+title: WMT Terminology 
+description: Terminology for understanding WMT results
```
Comment (cefoo) [inline events/wmt-terminology.md:4]: Hi dear Lia!

Here I would just say `Terminology used in WMT results`

**Review by cefoo — COMMENTED** [review]
Hi Lia!

Overall, I think it would be best to have shorter sentences. It is friendlier for MT and it simplify concepts.

I hope my comments are useful! Please take them as suggestions. :)

File: events/wmt-terminology.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+## TrueSkill
+
+TrueSkill is an adaptative model of competitions originally developed by Microsoft Research for the Xbox Live online gaming community.
+[WMT](wmt.md) developed a variation of the TrueSkill model adapted to the human evaluation of machine translation output.
+During [WMT15](wmt15.md) and [WMT16](wmt16.md), the human ranking for each task was produced using TrueSkill in the following manner:
```
[the remark this answers is not in the document]
Comment (bittlingmayer) [reply events/wmt-terminology.md:11]: "for all WMT tasks" sounds a bit better than "for every"

Also should be "translation tasks" - because there are other tasks like QE.

[the remark this answers is not in the document]
Comment (bittlingmayer) [reply events/wmt-terminology.md:11]: > [WMT](wmt.md) developed a variation of the TrueSkill model adapted to the human evaluation of machine translation output.

could be just

> [WMT](wmt.md) adapted TrueSkill to machine translation.

Although I'm not quite sure that "WMT" is really the agent here, maybe "WMT task organisers" or make it passive

> For WMT, TrueSkill was adapted to machine translation.

File: events/wmt-terminology.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+parent: Events
+title: WMT Terminology
```
Comment (bittlingmayer) [inline events/wmt-terminology.md:3]: Lowercase terminology because in our style guide title case is like that Not Like US Style

File: events/wmt-terminology.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## TrueSkill
+
+TrueSkill is an adaptative model of competitions originally developed by Microsoft Research for the Xbox Live online gaming community.
+[WMT](wmt.md) developed a variation of the TrueSkill model adapted to the human evaluation of machine translation output.
+During [WMT15](wmt15.md) and [WMT16](wmt16.md), the human ranking for each task was produced using TrueSkill in the following manner:
+
+First, 1,000 bootstrap-resampled runs are produced over all of the available data.
+A rank range for each system is then computed.
+To compute the rank range, the absolute rank of each system is collected in each fold and thrown out the top and bottom 2.5%.
```
Comment (bittlingmayer) [inline events/wmt-terminology.md:15]: "and thrown out .the top and bottom 2.5%" is ungrammatical here

chartac naxadasutyun a eli

File: events/wmt-terminology.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+There are two main types of scores produced by WMT: average scores and z-scores.
+The average z-score is used as the official ranking of the conference results.
+The z-score shows how many standard deviations below or above the mean an average or raw score is.
+
+To compute the z-score,
+1. Human assessment scores for translations are standardized according to each human assessor’s overall mean and standard deviation score.
+2. Average z-scores for individual segments belonging to a given system are computed.
+3. The final overall direct assessment score for that system is computed as the average standard of its segment scores.
+
+The average scores for systems are computed in the same way but without any score standardization applied.
```
Comment (bittlingmayer) [inline events/wmt-terminology.md:55]: You use American spellings here and in a few places

**Comment by bittlingmayer** [comment]
Could this just be a section in wmt.md?

In my opinion we should use 

<details>
<summary>Collapsible sections</summary>
like this
</details>

more.

(Unfortunately we can't use markdown inside of it, that's the main downside.)

If we really want a separate page, wmt/

Another option would be to start a general terminology/lexicon/glossary page.

**Comment by bittlingmayer** [comment]
If you don't hyperlink usages of these terms to the definitions, no one will ever find them.

In any case, I lean towards something like:

---
<details><summary><h2>Definitions</h2></summary>

### Z-score

</details>

at the bottom of the main page.

File: events/wmt-terminology.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+A rank range for each system is then computed.
+To compute the rank range, the absolute rank of each system is collected in each fold and thrown out the top and bottom 2.5%.
+Then systems are clustered into equivalence classes, containing systems with overlapping ranges.
+The ranges yield a partial ordering over systems at the 95% confidence level.
+
+## Relative ranking
+
+From 2014 to 2016, the relative ranking approach was the official scoring mechanism of the WMT conference. 
+
+In relative ranking, annotators rank five system outputs for a given segment in comparison to one another.
```
[the remark this answers is not in the document]
Comment (bittlingmayer) [reply events/wmt-terminology.md:23]: I think the original was clearer.

[the remark this answers is not in the document]
Comment (bittlingmayer) [reply events/wmt-terminology.md:23]: But too wordy

[the remark this answers is not in the document]
Comment (bittlingmayer) [reply events/wmt-terminology.md:23]: > For each input, annotators rank the outputs from five different systems.

**Comment by liashahnazaryan** [comment]
> If you don't hyperlink usages of these terms to the definitions, no one will ever find them.
> 
> In any case, I lean towards something like:
> 
> ## Definitions
> at the bottom of the main page.

We discussed this with @cefoo and decided to create a new file in the events directory without adding it to the nav and just linking to it in the wmt pages instead. So when creating the page I also added all the term hyperlinks in the wmt pages where the terms were used. 

I don't think starting a general terminology page would be the best solution here as there are only a few general terms that need explanation. What do you think, @bittlingmayer?

File: events/wmt-terminology.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+Overall system rankings are then produced by means of the TrueSkill algorithm. 
+
+The pairwise comparisons denote only relative ability between a pair of systems and cannot be used to infer their absolute quality. 
+
+## Direct assessment
+
+WMT16 added direct assessment scoring of system outputs as an investigatory ranking.
+Since [WMT17](wmt17.md), the approach has been the official scoring mechanism of the conference.
+
+In direct assessment, annotators provide a direct assessment of the quality of a single machine-translated output compared to a single reference, using an analog scale.
```
[the remark this answers is not in the document]
Comment (liashahnazaryan) [reply events/wmt-terminology.md:34]: > I would put the definition first, and then its history.

I wanted to save the structure the same throughout the article. For TrueSkill, the term definition is merged with its history, so I thought I would save that for other terms too. What do you think, @cefoo?

[the remark this answers is not in the document]
Comment (liashahnazaryan) [reply events/wmt-terminology.md:34]: > Also maybe just say what it is straighforwardly?
> 
> > **Direct assessment** is...
> > Direct assessment uses...

Direct assessment is just an evaluation method used by annotators, so considering your comment, I think about something like this: 
> Annotators use direct assessment to evaluate the quality of the system output relative to a reference translation.

What do you think?

File: events/wmt-terminology.md (hunk 1 above)
Comment (liashahnazaryan) [reply events/wmt-terminology.md:51]: I'm not sure about this one. `Standard deviation` is a general term in math, so if we define it, we should also define `mean` and maybe some other words or expressions. What do you think, @cefoo?

File: events/wmt-terminology.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+## Average score and z-score
+
+There are two main types of scores produced by WMT: average scores and z-scores.
+The average z-score is used as the official ranking of the conference results.
+The z-score shows how many standard deviations below or above the mean an average or raw score is.
+
+To compute the z-score,
+1. Human assessment scores for translations are standardized according to each human assessor’s overall mean and standard deviation score.
+2. Average z-scores for individual segments belonging to a given system are computed.
+3. The final overall direct assessment score for that system is computed as the average standard of its segment scores.
```
[the remark this answers is not in the document]
Comment (liashahnazaryan) [reply events/wmt-terminology.md:53]: I've changed the sentence a bit to make it less complicated. What about something like this?

> The average of all individual segments of a given system is computed as a z-score of that system.

**Review by liashahnazaryan — COMMENTED** [review]
I've made changes according to the suggestions and the overall discussion. I've also commented on several suggestions which aren't yet changed.

File: events/wmt-terminology.md (hunk 1 above)
Comment (cefoo) [reply events/wmt-terminology.md]: I agree, no need for the explanation.

File: events/wmt-terminology.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (cefoo) [reply events/wmt-terminology.md]: What about this? What do you think? Is it too far from reality?

```
2. The average z-scores of a given system are computed at the segment level.
3. With the segment-level results, the system-level average z-score is computed.
```

[the remark this answers is not in the document]
Comment (liashahnazaryan) [reply events/wmt-terminology.md]: yes, this is a lot better
I'll make changes

**Comment by zouharvi** [comment]
I really like this article. There are, however, two things mixed: (1) Human MT Evaluation and (2) description what WMT used when.

Maybe they could be detached so that when someone wants to read about _direct assessment_ they can go to the _Human MT Evaluation_ page and not _WMT Terminology_ one? The first article could also elaborate more on what the advantages and disadvantages of various approahces are, including examples.

**Comment by bittlingmayer** [comment]
Your other PR was merged - https://github.com/machinetranslate/machinetranslate.org/pull/283

So we can close this PR now!

**Comment by bittlingmayer** [comment]
Your other PR was merged - https://github.com/machinetranslate/machinetranslate.org/pull/283

So we can close this PR now!
